static/go/common/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Podman vanity import</title>
+    <meta name="go-import" content="podman.io/go/ git https://github.com/containers/container-libs.git">
+    <meta http-equiv="refresh" content="0; url=https://podman.io/">
```

Comment:
If I copy paste a module import path I generally like to get redirected to the code repo not the main website.